### PR TITLE
procurement inspector priority new field #134694633

### PR DESCRIPTION
**Pull request by nimaai**

### CODE REVIEW

File: engines/procurement/spec/features/csv.feature [hunk 1]
```diff
@@ -37,10 +38,43 @@ Feature: Exporting the data to a CSV-File
     Examples:
       | username  |
       | Barbara   |
-      | Roger     |
       | Hans Ueli |
 
   @csv
+  Scenario Outline: Export data
```
Comment (nimaai) [inline engines/procurement/spec/features/csv.feature]: @nadjaweisskopf we already have a scenario with the same name in this feature file. What is the purpose of this one?

File: engines/procurement/spec/features/csv.feature [hunk 2]
```diff
@@ -37,10 +38,43 @@ Feature: Exporting the data to a CSV-File
...
+      | Requested quantity         |
+      | Price                      |
+      | Total                      |
+      | Priority                   |
+      | Motivation                 |
+      | Supplier                   |
+      | Receiver                   |
+      | Point of Delivery          |
+      | State                      |
+    And the following fields are exported when the budget period has ended
```
Comment (nimaai) [inline engines/procurement/spec/features/csv.feature]: @nadjaweisskopf I don't understand this step. These two fields seem to be exported also for the current budget period (see lines 9, 26, 34).

Comment (nadjaweisskopf) [inline engines/procurement/spec/features/csv.feature]: Sorry, I copied the scenario. They shoudln't be called the same. The export for inpector's and requesters is not the same. The requesters are not allowed to see all data.

Comment (nimaai) [inline engines/procurement/spec/features/csv.feature]: Ok, now I understand. Thanks.

**Comment by nimaai** [comment]
@nadjaweisskopf if some field should not be exported, it means **not** that the respective column is not there in the exported file, but that the value is not displayed in that column if present. this is how it was already implemented and it makes sense, because there are conditions where the visibility of such field is not just depended on the role of the user but on some other value of the exported request, like e.g. `budget_period` or `category`.

**Pull request merged**